src/vcbuild/dmd.visualdproj

```diff
@@ -410,7 +410,7 @@
  </Config>
  <Folder name="dmd">
   <Folder name="gen">
-   <File tool="Custom" path="..\..\VERSION" customcmd="cmd /E:ON /Q /C &quot;for /f %%f in ( ..\..\VERSION ) do ( echo &quot;%%f&quot; )&quot; &gt;$(OutDir)\generated\verstr.h" uptodateWithSameTime="true" outfile="$(OutDir)\generated\verstr.h" />
+   <File tool="Custom" path="..\..\VERSION" customcmd="copy ..\..\VERSION $(OutDir)\VERSION" uptodateWithSameTime="true" outfile="$(OutDir)\VERSION" />
```

Comment:
Thanks for updating the VS project, too. 
Why still copy the file instead of using "..\.." as a string import path?